client/dev-tools/components/dev-tools.tsx

```diff
@@ -107,19 +100,6 @@ const DevTools = () => {
 		return;
 	}
 
-	const upgradeCtaCopy = hasEnTranslation(
```

Comment:
I believe this is currently behind a feature flag, so the translation check isn't necessary (either way, I've changed the strings for the reasons mentioned in the description!).

Reply:
While behind a feature flag, the check can be good if the feature releases before translations are complete. I think our timelines for release have shifted a couple times so the check may have been added anticipating a release before the new items were fully translated.

Reply:
Makes sense! I’d be happy to re-add them should they still be felt necessary once the strings are finalised.